storage/innobase/trx/trx0purge.cc

```diff
@@ -180,7 +180,7 @@ trx_purge_graph_build(sess_t* sess)
 	ut_ad(trx->sess == sess);
 
 	trx->id = 0;
-	trx->start_time = ut_time();
+	trx->init_start_time();
```

Comment:
Just curious: what problem does this solve? This purge pseudo-transaction does not really have a concept of start time.